variables.tf

```diff
@@ -1,21 +1,75 @@
-variable "lambda_name" {}
+variable "enabled" {
+  type        = "string"
+  default     = "true"
+  description = "Toggle the creation and destruction of all resources in this module"
+}
 
-variable "runtime" {}
+locals {
+  // transform the public `enabled` variable into a count integer
+  enabled = "${lower(var.enabled) == "true" ? 1 : 0}"
```

Comment:
This is not needed, because Terraform knows that "true" is the same as 1, and "false" is the same as 0.

Reply:
Yeah, but that is bad practice. Count should still be an explicit number.